utilities/dot-project/types.go

```diff
@@ -0,0 +1,136 @@
+package projects
+
+import (
+	"net/http"
+	"time"
+)
+
+type ProjectList []string
+
+type Project struct {
+	Name         string            `json:"name" yaml:"name"`
+	Description  string            `json:"description" yaml:"description"`
+	MaturityLog  []MaturityEntry   `json:"maturity_log" yaml:"maturity_log"`
+	Repositories []string          `json:"repositories" yaml:"repositories"`
+	Social       map[string]string `json:"social" yaml:"social"`
+	Artwork      string            `json:"artwork" yaml:"artwork"`             // Artwork URL
+	Website      string            `json:"website" yaml:"website"`             // Project website URL
+	MailingLists []string          `json:"mailing_lists" yaml:"mailing_lists"` // Mailing lists for project
+	Audits       []Audit           `json:"audits" yaml:"audits"`               // Security audits for project
+
+	// New fields merged from project.toml
+	SchemaVersion string               `json:"schema_version,omitempty" yaml:"schema_version,omitempty"`
+	Type          string               `json:"type,omitempty" yaml:"type,omitempty"`
+	Security      *SecurityConfig      `json:"security,omitempty" yaml:"security,omitempty"`
+	Governance    *GovernanceConfig    `json:"governance,omitempty" yaml:"governance,omitempty"`
+	Legal         *LegalConfig         `json:"legal,omitempty" yaml:"legal,omitempty"`
+	Documentation *DocumentationConfig `json:"documentation,omitempty" yaml:"documentation,omitempty"`
+}
+
+type PathRef struct {
+	Path string `json:"path" yaml:"path"`
+}
+
+type SecurityConfig struct {
+	Policy      *PathRef `json:"policy,omitempty" yaml:"policy,omitempty"`
+	ThreatModel *PathRef `json:"threat_model,omitempty" yaml:"threat_model,omitempty"`
+}
```

Comment:
```suggestion
type SecurityConfig struct {
	Contact     *PathRef `json:"contact,omitempty" yaml:"contact,omitempty"`
	Policy      *PathRef `json:"policy,omitempty" yaml:"policy,omitempty"`
	ThreatModel *PathRef `json:"threat_model,omitempty" yaml:"threat_model,omitempty"`
}
```

Besides policy, it'd be good to have a general security contact, like email to report vulnerabilities to. Doesn't necessarily have to be called contact.

Also what actually is threatmodel?  I don't recall it in insights or baseline - might be wrong though

Reply:
email - those are whats generally used for security reports, even if its the maintainer list

Reply:
+1 to email, but at some point it would be useful to support other fields but I think having email as primary/required is useful.

Not to derail, but some folks actually use external services for things like security contacts, and so something like one or more contacts is sort of a moot point. However, I don't think you need to support that on the first pass.

Reply:
Baseline does have a requirement for a threat model for Baseline L3:

> - [ ] **OSPS-SA-03.02**: When the project has made a release, the project MUST perform a threat modeling and attack surface analysis to understand and protect against attacks on critical code paths, functions, and interactions within the system.